src/Context/ViewContext.js

```diff
@@ -0,0 +1,13 @@
+import React, { createContext, useState } from 'react'
+
+export const ViewContext = createContext()
+
+export const ViewProvider = ({ children }) => {
+  const [ view, setView ] = useState('timer')
+
+  return (
+    <ViewContext.Provider value={[ view, setView ]}>
+      { children}
+    </ViewContext.Provider>
+  )
+}
```

Comment:
Great work with the ViewContext!! This is so great for how we're cycling through components.